### PR TITLE
[PREVIEW] Rdm 2654- Update User Roles

### PR DESCRIPTION
**Pull request by mohammedanas**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

https://tools.hmcts.net/jira/browse/RDM-2654

As a CMCA, I want to the ability to Update an existing user roles in ccd for a selected/assigned jurisdiction.
Fields:
Role (readonly)
Security Classification

The Clasification States are: PUBLIC PRIVATE RESTRICTED
and the default to user roles

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/views/user-roles.html [hunk 1]
```diff
@@ -53,11 +53,11 @@
...
     <tr >
       <td>
-      <form id="userprofile_{{loop.index}}" method="post" action="/updateuserrole" >
+      <form id="userrole_{{loop.index}}" method="post" action="/updateuserroleform" >
         {{ csrfProtection(csrfToken) }}
         <input type="hidden" name="role" value="{{userrole.role}}" />
         <input type="hidden" name="classification" value="{{userrole.security_classification}}" />
-        <div>{{userrole.role}}</div>
+         <a href="#" onclick="document.getElementById('userrole_{{loop.index}}').submit();"><div>{{userrole.role}}</div></a>
       </form>
```
Comment (adr1ancho) [inline src/main/views/user-roles.html]: It seems you might have 1 space too many

Comment (mohammedanas) [reply src/main/views/user-roles.html]: Not with you, can please explain a bit more

Comment (Daniel-Lam) [reply src/main/views/user-roles.html]: @mohammedanas - there is an extra space before `<a href="#"...`. Please remove

File: src/main/views/user-roles/create-user-roles.html [hunk 2]
```diff
@@ -47,14 +47,21 @@ <h2 class="heading-large padding">Create User Roles</h2>
...
 
-    <form id="createuserroleform" name="createuserform" method="POST" action="/createuserrole">
+    <form id="createuserroleform" name="createuserform" method="POST" action="{{submitUserRoleEndPoint}}">
     
       {{ csrfProtection(csrfToken) }}
         <div class="form-group">
+        
             <label class="form-label" for="userroles">User Roles</label>
-            <input type="text" class="form-control" id="role" name="role" maxlength="100"/><br><br> 
+             {%if update == true%}
```
Comment (Daniel-Lam) [inline src/main/views/user-roles/create-user-roles.html]: Slightly strange indentation here - 1 space before `{%if update == true}`? Should line up with `{% endif %}` on Line 62

File: src/main/views/user-roles/create-user-roles.html [hunk 3]
```diff
@@ -47,14 +47,21 @@ <h2 class="heading-large padding">Create User Roles</h2>
...
+                 <input type="hidden" id="role" name="role" value="{{role}}" />
+                 <label class="form-label-bold" >{{role}}</label><br><br>
+                 <input class="form-control" id="update" name="update" type="hidden" value="{{update}}"/>
+             {% else %}
```
Comment (Daniel-Lam) [inline src/main/views/user-roles/create-user-roles.html]: Slightly strange indentation here - should line up with `{% endif %}` on Line 62

File: src/main/views/user-roles.html (hunk 1 above)
Comment (mohammedanas) [reply src/main/views/user-roles.html]: addressed

File: src/main/views/user-roles/create-user-roles.html (hunk 2 above)
Comment (mohammedanas) [reply src/main/views/user-roles/create-user-roles.html]: addressed

File: src/main/views/user-roles/create-user-roles.html (hunk 3 above)
Comment (mohammedanas) [reply src/main/views/user-roles/create-user-roles.html]: addressed

**Pull request merged**